examples/service-clients/odsp-client/shared-tree-demo/.npmignore

```diff
@@ -1,5 +1,6 @@
 nyc
 *.log
 **/*.tsbuildinfo
-dist/tests
+dist/test
```

Comment:
This package doesn't have tests. Updated to use the standard paths for test file exclusions, for when it has any tests.